aries-site/src/data/structures/components.js

```diff
@@ -1850,6 +1850,7 @@ export const components = [
   },
   {
     name: 'DataSort',
+    accessibility: 'Passed WCAG 2.2 AAA',
```

Comment:
Looks like we are meeting AA not AAA
```suggestion
    accessibility: 'Passed WCAG 2.2 AA',
```